src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,123 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  min-height: 406px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+
+  &__pic {
+    background-repeat: no-repeat;
+    object-fit: fill;
+    object-position: center;
+    width: 160px;
+    height: 134px;
+
+    &--position {
+      @extend %margin-0-auto;
+    }
+
+  }
+
+  &__header {
+    @include text-format
+      ($font-size-medium,
+      $font-weight-medium,
+      $line-height-medium,
+      $color-font-main
+    );
+
+    &--position {
+      margin-top: 40px;
+    }
+  }
+
+  &__code {
+    @include text-format
+      ($font-size-regular,
+      $font-weight-regular,
+      $line-height-regular,
+      $color-font-secondary
+    );
+
```

Comment:
```suggestion
    @include text-format(
      $font-size-regular,
      $font-weight-regular,
      $line-height-regular,
      $color-font-secondary
    );

```
everywhere

Reply:
Thank you. I`ll try to be more attentive